packages/react-dom/src/client/ReactDOMFiberSelect.js

```diff
@@ -21,7 +23,7 @@ if (__DEV__) {
 
 type SelectWithWrapperState = HTMLSelectElement & {
   _wrapperState: {
-    initialValue: ?string,
+    initialValue: ?ToStringValue,
```

Comment:
It's curious that this is nullable and it's not clear to me where `initialValue` is even used. 

Not important for this PR, but I can't help but wonder if `initialValue` can go away.

Reply:
That's a good point. I'll try and play around with it and see what I find out 🙂Feel free to report an issue, regardless.